sdk/common/src/main/java/io/opentelemetry/sdk/internal/AttributesMap.java

```diff
@@ -47,10 +46,10 @@ public static AttributesMap create(long capacity, int lengthLimit) {
   /** Add the attribute key value pair, applying capacity and length limits. */
   public <T> void put(AttributeKey<T> key, T value) {
```

Comment:
Because `AttributeMap extends HashMap`, a caller was accidentally bypassing this method and calling `Map#put(T, V)` directly, which was preventing the application of attribute limits.

We protect against this by delegating to `HashMap` instead of extending.